### ISSUE TITLE
Error GETing https://del.icio.us/ (1.81)

### ISSUE THREAD

**Issue by eserte** [libwww-perl/WWW-Mechanize#54]
On smoker systems without LWP::Protocol::https installed I see the following failure:
```
Error GETing https://del.icio.us/: Protocol scheme 'https' is not supported (LWP::Protocol::https not installed) at t/live/encoding.t line 26.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 2 just after 2.
t/live/encoding.t ........ 
Dubious, test returned 2 (wstat 512, 0x200)
All 2 subtests passed 
```

**Comment by karenetheridge** [comment]
a `use Test::Needs 'LWP::Protocol::https;` should clear that up.

**Comment by oalders** [comment]
@eserte thanks very much! Closed via c6e324cd3f4
